pkg/volume/util.go

```diff
@@ -193,6 +193,15 @@ func GenerateVolumeName(clusterName, pvName string, maxLength int) string {
 	return prefix + "-" + pvName
 }
 
+// Check if the path from the mounter is empty.
+func GetPath(mounter Mounter) (string, error) {
+	path := mounter.GetPath()
+	if path == "" {
+		return "", fmt.Errorf("Path is empty")
```

Comment:
Ack. I'll print the type of the mounter. At this point we can get to just either the VolumeInfo or the Mounter only from what i can tell.